ui/components/app/selected-account/index.scss

```
@@ -43,11 +43,11 @@
     background-color: unset;
 
     &:hover {
-      background-color: var(--Grey-000);
+      background-color: var(--color-background-alternative);
     }
 
     &:active {
-      background-color: #d9d7da;
+      background-color: var(--color-background-default);
```

Just keep it as alternative
```suggestion
      background-color: var(--color-background-alternative);
```